README.md

```diff
@@ -246,6 +248,8 @@ Translations of the guide are available in the following languages:
   As far as embedded expressions go, there are also two acceptable
   options:
 
+  **Skookum uses the first option, no spaces in string embedded expressions.**
```

Comment:
I think we all like spaces right? I changed the guide to reflect it, unless I am misunderstanding someone.